applications/vanilla/views/discussion/tag.php

```diff
@@ -0,0 +1,16 @@
+<?php if (!defined('APPLICATION')) exit(); ?>
+    <h1><?php echo $this->data('Title'); ?></h1>
+<?php
+/** @var Gdn_Form $form */
+$form = $this->Form;
+echo $form->open(['id' => 'DiscussionAddTagForm']);
+echo $form->errors();
+
+echo '<div class="Form-Tags P">';
+    // Tag text box
+    echo $this->Form->label('Tags', 'Tags');
+    echo $this->Form->textBox('Tags', array('data-tags' => ''));
+echo '</div>';
+
+?>
+<?php echo $this->Form->close('Save');
```

Comment:
This view is functional, but very minimal. It feels a bit cramped. Maybe add a simple sentence indicating what is being done (e.g. "Which tags would you like to add to the discussion?") and making the input box a bit wider. The design of these windows is all over the board, but some have an "OK" and a "Close" button with the "OK" button having the `PrimaryButton` class, allowing it to be styled a bit differently.